### PR TITLE
fix: page height

### PR DESCRIPTION
**Pull request by banahaker**
# [ BUG FIXING ] MainView page height

change MainView Component height from `100%` to `auto`

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by SiongSng — CHANGES_REQUESTED** [review]
extra white space

**Comment by banahaker** [comment]
why 100%

**Comment by banahaker** [comment]
i change height of MainView component to `auto`.

File: src/components/BlockHomeIntro.vue [hunk 1]
```diff
@@ -57,6 +57,9 @@ const modules = [Pagination];
 .intro {
   padding: 20px 0;
   width: 100%;
+  padding: 45px;
+  height: calc(100% - $FooterHeight - $HeaderHeight);
+  // footer height: 127px; header height: 80px
```
Comment (SiongSng) [inline src/components/BlockHomeIntro.vue:62]: remove the comment

**Review by SiongSng — CHANGES_REQUESTED** [review]

File: src/components/BlockHomeIntro.vue (hunk 1 above)
Comment (banahaker) [reply src/components/BlockHomeIntro.vue:62]: what? is something wrong here?

Comment (SiongSng) [reply src/components/BlockHomeIntro.vue:62]: remove   `// footer height: 127px; header height: 80px`

**Review by SiongSng — APPROVED** [review]
LGTM

**Pull request merged**